### PR TITLE
BO: Fixed bug when changing impact on price (#BOOM-421)

### PR DESCRIPTION
**Pull request by antho-girard**
| Questions | Answers |
| --- | --- |
| Branch? | develop |
| Description? | Version 1.7 Alpha 4. All browsers. Fixed the issue when entering a value for impact on price (tax incl. and tax excl.). If you modify tax incl. then it also updates tax excl. and vice versa. |
| Type? | bug fix |
| Category? | BO |
| BC breaks? | no |
| Deprecations? | no |
| Fixed ticket? | http://forge.prestashop.com/browse/BOOM-421 |
| How to test? | On product page, combination tab, edit an existing one and enter a value in "Impact on price" field tax excl. or tax incl. It will now calculate (previously ok) and update the other one. |

#### Important guidelines
- Make sure [your local branch is up to date](https://help.github.com/articles/syncing-a-fork/) before commiting your changes!
- Your code MUST respect [our Coding Standards](http://doc.prestashop.com/display/PS16/Coding+Standards) (for code written in PHP, JavaScript, HTML/CSS/Smarty/Twig, SQL)!
- Your commit name MUST respect our [naming convention](http://doc.prestashop.com/display/PS16/How+to+write+a+commit+message)!

### CODE REVIEW

File: admin-dev/themes/default/js/bundle/product/form.js [hunk 1]
```diff
@@ -1622,7 +1622,7 @@ var priceCalculation = (function() {
     },
     'impactTaxInclude': function(obj) {
       var price = parseFloat(obj.val().replace(/,/g, '.'));
-      var targetInput = obj.parent().parent().parent().find('input.attribute_priceTI');
+      var targetInput = obj.parent().parent().parent().parent().find('input.attribute_priceTI');
```
Comment (mickaelandrieu) [inline admin-dev/themes/default/js/bundle/product/form.js]: any chance to find a better selector ? :)

Comment (xBorderie) [inline admin-dev/themes/default/js/bundle/product/form.js]: I talked to @antho-girard about this on Gitter, he'll try to find a more durable fix :)

**Comment by antho-girard** [comment]
selector is updated :)

**Comment by mickaelandrieu** [comment]
Thanks!

**Comment by xBorderie** [comment]
Are we using jQuery's closest() or the DOM's closest() here? 'cos if it's the DOM's, it's not quite ready for primetime: [not supported by IE & Edge](http://caniuse.com/#feat=element-closest).

**Comment by Shudrum** [comment]
Thank you for the alert @xBorderie, and yep, obj is a jQuery object 👍 

But … wow …
```javascript
.parent().parent().parent().
```
:cry::gun:

**Comment by xBorderie** [comment]
giphy

**Pull request merged**